nextjs-logs-manager/app/db/controller/userController.ts

```
@@ -0,0 +1,17 @@
+  static async listUsers() {
+    return await getUsers();
+  }
+
+  static async getUser(input: string) {
+    return await getUserByUsernameOrEmail(input);
+  }
+
+  static async addUser(username: string, email: string, password: string, role: string) {
+    return await createUser(username, email, password, role);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)